add_pypi_package.sh

```diff
@@ -39,18 +41,14 @@ generate_pypi_package() {
   else
     RPM_NAME_ARG=""
   fi
-  pyp2rpm --no-autonc -s -d $PACKAGE_DIR -v $VERSION $RPM_NAME_ARG $PYPI_NAME
+  pyp2rpm --no-autonc -s -t $TEMPLATE -b $BASE_PYTHON -d $PACKAGE_DIR -v $VERSION $RPM_NAME_ARG $PYPI_NAME
   echo "FINISHED"
   if [[ $UPDATE == true ]]; then
     echo "Restoring changelogs..."
     cat OLD_CHANGELOG >> $PACKAGE_DIR/*.spec
     sed -i '/^%changelog/,/^%changelog/{0,//!d}' $PACKAGE_DIR/*.spec
     rm OLD_CHANGELOG
-    CHANGELOG="- Update to $VERSION"
```

Comment:
Isn't this part still useful?

Reply:
`pyp2rpm` currently does not support setting the changelog message, so this would generate two entries. I'd like to fix `pyp2rpm`, but not right now.

Reply:
So you end up with the old changelog and a new 'Initial package` entry that you need to manually modify?

Reply:
at the moment: yes